.github/workflows/main.yml

```diff
@@ -242,6 +242,36 @@ jobs:
           name: code-coverage-report-unit-aries-vcx
           path: /tmp/artifacts/coverage
 
+  #TODO - can this be included within code-coverage-aries-vcx-integration-tests?
```

Comment:
@Patrik-Stas any thoughts on this? not too familiar with best practices for your github pipelines

Reply:
as discussed on today's aries-vcx call, good for now, we'll refactor coverege handling across the board in CI as separate task